From 762717181fbefb9e8fe3739e13ad907dddebc877 Mon Sep 17 00:00:00 2001
From: Matt Turner <msturner@google.com>
Date: Thu, 11 Jul 2024 15:06:06 +0000
Subject: [PATCH] CHROMIUM: remove post_install hook

The post_install hook adds an unconditional dependency on
gio-querymodules, which would require dev-libs/glib to be added to
BDEPEND.

But the hook is not actually executed when DESTDIR is set, which it
always is under portage.
---
 meson.build | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/meson.build b/meson.build
index a774ea6..5703009 100644
--- a/meson.build
+++ b/meson.build
@@ -146,9 +146,6 @@ subdir('po')
 subdir('proxy')
 subdir('tls')
 
-# Will automatically pick it up from the cross file if defined
-gnome.post_install(gio_querymodules: gio_module_dir)
-
 summary({
   'GnuTLS support': giomodules.contains('giognutls'),
   'OpenSSL support': giomodules.contains('gioopenssl'),
-- 
2.45.2.803.g4e1b14247a-goog

